crates/sqllib/src/timestamp.rs

```diff
@@ -813,6 +814,36 @@ pub fn hop_TimestampN_ShortInterval_ShortInterval_ShortInterval(
     }
 }
 
+#[doc(hidden)]
+pub fn parse_timestamp__(format: String, st: String) -> Option<Timestamp> {
```

Comment:
Is this one of those cases where you could pass strings by reference, as `&str`?

Reply:
So far I am never passing strings by reference, but would be glad to discuss a design that would be general.